### PR TITLE
feat: emotes

### PR DESCRIPTION
**Pull request by cazala**
Adds schemas related to emotes

### CODE REVIEW

File: test/emote-category.spec.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import expect from 'expect'
+import { EmoteCategory } from '../src'
+import { testTypeSignature } from './test-utils'
+
+describe('EmoteCategory tests', () => {
+  const wearableCategory: EmoteCategory = EmoteCategory.SIMPLE
```
Comment (nachomazzara) [inline test/emote-category.spec.ts:6]: It should be `emoteCategory`?

File: src/dapps/nft.ts [hunk 2]
```diff
@@ -189,7 +196,13 @@ export namespace NFT {
                 type: 'boolean'
               }
             },
-            required: ['bodyShapes', 'category', 'description', 'rarity'],
+            required: [
+              'bodyShapes',
+              'category',
+              'description',
+              'rarity',
+              'isSmart'
```
Comment (nachomazzara) [inline src/dapps/nft.ts:204]: Is it ok to add `isSmart` here?

File: test/nft.spec.ts [hunk 3]
```diff
@@ -187,4 +188,43 @@ describe('NFT tests', () => {
...
+        emote: {
+          description: 'Some Description',
+          category: EmoteCategory.SIMPLE,
+          bodyShapes: [BodyShape.MALE, BodyShape.FEMALE],
+          rarity: Rarity.MYTHIC
+        }
+      },
+      issuedId: '16',
+      itemId: '0',
+      category: NFTCategory.ESTATE,
```
Comment (nachomazzara) [inline test/nft.spec.ts:214]: Is it ok to use the `ESTATE` category?

File: src/dapps/nft.ts (hunk 2 above)
Comment (cazala) [reply src/dapps/nft.ts:204]: yes, it was missing, but it should be required

File: test/emote-category.spec.ts (hunk 1 above)
Comment (cazala) [reply test/emote-category.spec.ts]: yep, changed ✅

File: test/nft.spec.ts (hunk 3 above)
Comment (cazala) [reply test/nft.spec.ts]: yes, updated ✅

**Review by nachomazzara — APPROVED** [review]
Great

**Pull request merged**